frontend/public/components/search.tsx

```diff
@@ -28,17 +29,20 @@ const ResourceList = connectToModel(({ kindObj, mock, namespace, selector, nameF
   const ns = kindObj.namespaced ? namespace : undefined;
 
   return (
-    <AsyncComponent
-      loader={componentLoader}
-      namespace={ns}
-      selector={selector}
-      nameFilter={nameFilter}
-      kind={kindObj.crd ? referenceForModel(kindObj) : kindObj.kind}
-      showTitle={false}
-      hideTextFilter
-      autoFocus={false}
-      mock={mock}
-    />
+    <div>
+      <div className="co-search-group__float-right">{getBadgeFromType(kindObj.badge)}</div>
```

Comment:
This is probably better to handle in the `FireMan_` component (list-page.tsx). I suspect the badge is not showing up already since we hide the title.

Reply:
The badge is already in the PageHeader which we hide.  This solution adds the badge to just the search page results for now.  The next steps for the search page involve sreious updates to the list-page file and so I thought we could have this as a temporary solution until 4.5.

Reply:
@spadgett - can we go ahead and accept this fix for now with the understanding that ListPage will be re-worked in the next release?

Reply:
@zherman0 This should be a tiny change to the existing list page component. I think it's less code than you have here.